python/lsst/sims/maf/plots/spatialPlotters.py

```diff
@@ -106,7 +106,7 @@ def __call__(self, metricValueIn, slicer, userPlotDict, fignum=None):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
             cb = plt.colorbar(im, shrink=0.75, aspect=25, orientation='horizontal',
-                            extend='both', extendrect=True, format=plotDict['cbarFormat'])
+                              format=plotDict['cbarFormat'], extendrect=True)
```

Comment:
Using extendrect instead of extend='both' here, we should probably also drop it at line 538.